lib/server.js

```diff
@@ -72,7 +74,10 @@ var Server = function (cliOptions, done) {
     reporter: ['factory', reporter.createReporters],
     capturedBrowsers: ['type', BrowserCollection],
     args: ['value', {}],
-    timer: ['value', {setTimeout: setTimeout, clearTimeout: clearTimeout}]
+    timer: ['value', {
+        setTimeout: function () { return setTimeout.apply(root, arguments)},
+        clearTimeout: function (timeoutId) { clearTimeout(timeoutId)}
```

Comment:
setTimeout has variables args, whereas clearTimeout only ever receives a single arg. I can change it if you think this is less consistent.